tests/restarting/from_7.0.0/UpgradeAndBackupRestore-1.toml

```diff
@@ -0,0 +1,55 @@
+[[test]]
+testTitle = 'SubmitBackup'
+simBackupAgents= 'BackupToFile'
+clearAfterTest = false
+runConsistencyCheck=false
+
+    [[test.workload]]
+    testName = 'SubmitBackup'
+    delayFor = 0
+    stopWhenDone = false
+
+[[test]]
+testTitle = 'FirstCycleTest'
+clearAfterTest=false
```

Comment:
Should this be `clearAfterTest=true` so that the restore in the second part is not redundant (?) I may be misunderstanding how the `SaveDatabase` portion works though

Reply:
The `RestoreBackup` workload clears the database before running the restore. If we cleared here while the backup is still running, the clear mutation would be part of the backup, so the restored database would be empty.